template/src/core/app/App.tsx

```diff
@@ -0,0 +1,38 @@
+import {lazy, Suspense} from "react";
+import {Switch, Route} from "react-router-dom";
+
+import AboutPage from "../../about/AboutPage";
+import RouteLoading from "../route/loading/RouteLoading";
+import ROUTES from "../route/routes";
+
+const HomePage = lazy(
+  () => import(/* webpackChunkName: "home-page" */ "../../home/HomePage")
+);
+const NotFoundPage = lazy(
+  () =>
+    import(
+      /* webpackChunkName: "not-found-page" */ "../route/not-found-page/NotFoundPage"
+    )
+);
+
+function App() {
+  return (
+    <Suspense fallback={<RouteLoading />}>
```

Comment:
It is good to have also ErrorBoundary component at least at the top level.
[error-boundary](https://reactjs.org/docs/error-boundaries.html)

Reply:
I'm not sure why, but we don't have it in the latest projects, I'll check it and we can discuss on the team meeting